### PR TITLE
Add defaults and update k8s dev version

### PR DESCRIPTION
**Pull request by gouthamMN**
### Which issue this PR addresses:

Fixes: 

- https://issues.redhat.com/browse/ARO-4576
- https://issues.redhat.com/browse/ARO-4514

### What this PR does / why we need it:

- Update the aro-rp az aro extension (_params.py) with defaults
- Update K8s version in development.

### Test plan for issue:
- Existing E2E tests

### Is there any documentation that needs to be updated for this PR?
No

### CODE REVIEW

File: python/az/aro/azext_aro/_params.py [hunk 1]
```diff
@@ -60,45 +60,45 @@ def load_arguments(self, _):
...
         c.argument('pod_cidr',
-                   help='CIDR of pod network. Must be a minimum of /18 or larger.',
+                   help='CIDR of pod network. Must be a minimum of /18 or larger. [Default: 10.128.0.0/14]',
                    validator=validate_cidr('pod_cidr'))
         c.argument('service_cidr',
-                   help='CIDR of service network. Must be a minimum of /18 or larger.',
+                   help='CIDR of service network. Must be a minimum of /18 or larger. [Default: 172.30.0.0/16]',
                    validator=validate_cidr('service_cidr'))
         c.argument('outbound_type',
                    help='Outbound type of cluster. Must be "Loadbalancer" (default) or "UserDefinedRouting".',
```
Comment (0kashi) [inline python/az/aro/azext_aro/_params.py:69]: can we make this follow the same format for conveying the default values? Something like:

> Outbound type of cluster. Allowed values: Loadbalancer, UserDefinedRouting. [Default: Loadbalancer]

File: python/az/aro/azext_aro/_params.py [hunk 2]
```diff
@@ -60,45 +60,45 @@ def load_arguments(self, _):
...
                    validator=validate_outbound_type)
         c.argument('enable_preconfigured_nsg', arg_type=get_three_state_flag(),
-                   help='Use Preconfigured NSGs. Allowed values: false, true.')
+                   help='Use Preconfigured NSGs. Allowed values: false, true. [Default: false]')
         c.argument('disk_encryption_set',
-                   help='ResourceID of the DiskEncryptionSet to be used for master and worker VMs.',
+                   help='ResourceID of the DiskEncryptionSet to be used for master and worker VMs. [Default: false]',
```
Comment (0kashi) [inline python/az/aro/azext_aro/_params.py:74]: Does a value of "false" make sense here?

**Comment by 0kashi** [comment]
added a few comments.

**Review by bennerv — APPROVED** [review]
lgtm

**Comment by bennerv** [comment]
I'm removing next-release label as this isn't coupled to the RP release.

**Review by kimorris27 — APPROVED** [review]

**Review by s-amann — APPROVED** [review]

**Pull request merged**